EventFilter/Utilities/test/mergeStreamerFile_cfg.py

```diff
@@ -0,0 +1,59 @@
+import FWCore.ParameterSet.Config as cms
+from argparse import ArgumentParser
+
+parser = ArgumentParser(description='Test merge using GlobalEvFOutputModule')
+parser.add_argument("--input", action="append", default=[], help="Input files")
+parser.add_argument("--numThreads", help="number of threads to use", type=int, default=3)
+parser.add_argument("--numFwkStreams", help="number of cmsRun streams", type=int, default=2)
+parser.add_argument("--changeBranchIDLists", help="modify the branchIDLists", type=bool, default=False)
```

Comment:
This `--changeBranchIDLists` option seems to be unused in this configuration.

Reply:
That is true. Do I need to change it in this PR?